docs/pvc-recovery.md

```diff
@@ -0,0 +1,263 @@
+# Prior PVC data recovery
+
+## Why
+
+In the case where you had a cluster that was running, but somehow ended up with a new cluster backed by a new PVC (typically caused by the elasticsearch CR being deleted and then recreated). You may want to recover that data from your past cluster.
+
+This guide seeks to walk users through the steps necessary to reindex data from their past cluster(s) to a currently running one.
+
+## How
+
+This process involves using the `_reindex` api made available by ElasticSearch, specifically the [https://www.elastic.co/guide/en/elasticsearch/reference/6.8/reindex-upgrade-remote.html](reindex from a remote cluster) procedure.
+
+To begin, this guide assumes you have a currently running cluster in the `openshift-logging` namespace with the cluster name `elasticsearch` (this is also the name of your elasticsearch CR).
+
+1. Update your `elasticsearch` CR to be `Unmanaged`
+   - We will be updating it's configmap, and we don't want the Elasticsearch Operator reverting those changes until we're done recovering.
+
+1. Create a local elasticsearch CR for your recovery cluster
+   - There is an example in the [#Appendix](appendix)
+   - You may need to decrease the resources for this new cluster so that you can ensure the pods will be deployed.
+   - Ensure that the following match your existing cluster:
+     - nodeCount and roles
+     - redundancyPolicy
```

Comment:
Ahh... we are using reindex API :-)nwm

Reply:
yeah, i'm not sure if it makes a huge impact since the operator isn't going to be doing anything to the cluster while it has the data but i was thinking it might be good for it to match up just in case.